examples/foundry/README.md

```diff
@@ -0,0 +1,68 @@
+# Foundry example for Sapphire developers
+
+This project showcases Sapphire development with Foundry. 
+
+
+## Overview
+
+Test folder contains a set of Forge tests that access precompiles 
+and demonstrate sapphire-specific . 
+Makefile contains build/test/clean targets.
+
+
+The project provides several key precompiles:
+
+### Cryptographic Operations
+- `RandomBytesPrecompile`: Generate random bytes
+- `X25519DerivePrecompile`: Derive shared secrets using X25519
+- `DeoxysiiSealPrecompile`: Encrypt data using Deoxys-II
+- `DeoxysiiOpenPrecompile`: Decrypt data using Deoxys-II
+- `Curve25519ComputePublicPrecompile`: Compute public keys
+
+### Key Management
+- `KeypairGeneratePrecompile`: Generate cryptographic keypairs
+- `SignPrecompile`: Sign messages
+- `VerifyPrecompile`: Verify signatures
+
+### Consensus Operations
+- `SubcallPrecompile`: Enhanced version with CBOR parsing and 
+state management for:
+  - Delegations
+  - Undelegations
+  - Receipt tracking
+
+## Key Features
+
+1. **Sapphire precompiles as contracts**
+   - Can run as native precompiles
+   - Easy import into forge tests
+
+2. **Decryption Base contract**
+   - Enables decryption at contract level
+   - Used as a base contract for other contracts that implement encryption
+
```

Comment:
```suggestion
```

This is just copy&paste from the integrations right? No need to put it inside the examples as well.

Reply:
Updated README